cmd/admin-kms-key.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * MinIO Client (C) 2017, 2018 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import "github.com/minio/cli"
+
+var adminKMSKeyCmd = cli.Command{
+	Name:   "key",
+	Usage:  "manage KMS keys",
```

Comment:
to make it clear that this is only for listing and key-rotation and not for other management operations like key creation/deletion ,the Usage should just list supported key management operations,imo

Reply:
We can clarify what KMS operations are supported. However, the list `status, listing, rotation` is not comprehensive. It depends on which future requirements we get in the future. We can cover a lot with these 3 operations but depending on user requirements we may extend that - e.g. disabling a specific key.

But we can extend that list if we add new functionality - we just must not forget that then.